magicmarie/book_a_meal
# UItemplates

@jokamjohn @mutebironald4
#### What's this PR do?
This adds UI templates as it makes use of the web pack and Sass files. Now web pack can bundle modules.

#### Where should the reviewer start?
CSS and Html changes made in the UItemplates branch. Use yarn watch to compile the CSS incase you change any CSS code.

#### How should this be tested?
Check out https://magicmarie.github.io/index.html
To view as Admin, got to https://magicmarie.github.io/admin.html 
To view as the customer, go to https://magicmarie.github.io/customer.html 


#### Questions:
Do you think the User Interface is User-friendly??
What would you suggest I improve??
Any functionality rendered important that I may be missing?
Any future improvements to a making a pull request template?
